docs/components/frame.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebarDepth: 0
+---
+
+# Frame
+
+`<Frame>` is a UI component used to display `<Page>` elements.
+
+A frame with a default page
+
+```html
```

Comment:
there should be a live demo section that lets user to modify all attributes of the component